### ISSUE TITLE
Bug 59185602: [React Native Gallery Preview]: No success info while invoking 'counter example A touchable without feedback' button present under "A touchable without feedback counter" group.

### ISSUE THREAD

**Issue by vineethkuttan** [microsoft/react-native-gallery#661]
Build Details:
OS Version: 24H2 
OS Build: 26100.5074
App: React Native Gallery Preview
AT Used: Narrator

Pre-Requisite:
Turn on Narrator (Ctrl + Win + Enter).

Repro-Steps:
Launch the React Native Gallery-Preview app.
Navigate to the 'Expand menu' button and invoke it.
Navigate to 'All samples’ button present in left navigation pane and invoke it.
Navigate to 'Touchable Without Feedback' button present under “Legacy” group and invoke it.
Navigate to 'counter example A touchable without feedback' button present under "A touchable without feedback counter" group. button and invoke it.
Now listen to the Narrator announcement.
Actual Result:
While invoking the 'counter example A touchable without feedback' button, number will be increasing inside the button but narrator doesn’t announce about the number.

Expected Result:
Narrator should announce about the available number present inside the 'counter example A touchable without feedback' button.
 
User impact:
Screen Reader users will not be able to know the value present inside the button if narrator fails to read the number.

MAS Reference Link: [MAS 4.2.12 – Change Notification](https://aka.ms/MAS4.2.12)

Trap ID:[1.9 - Feedback Failure](https://aka.ms/FeedbackFailure)

“Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment section.”
